### PR TITLE
Feat sergio

### PR DESCRIPTION
**Pull request by gsricco**

### CODE REVIEW

File: src/entities/postsList/api/getPosts.api.types.tsx [hunk 1]
```diff
@@ -18,9 +18,14 @@ export type GetPostsQuery = {
...
       description: string
-      images: Array<{ __typename?: 'ImagePost'; height: number; url: string; width: number }>
+      images?: Array<{
+        __typename?: 'ImagePost'
+        height?: null | number
+        url?: null | string
+        width?: null | number
+      }> | null
       ownerId: number
     }>
```
Comment (nazarPryt) [inline src/entities/postsList/api/getPosts.api.types.tsx:30]: some comment

File: src/entities/paymentsList/api/getAllPayments.api.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
 import { gql } from '@apollo/client'
-export const GET_ALL_PAYMENTS = gql`
+export const GET_PAYMENTS = gql`
   query getAllPayments(
     $pageSize: Int
     $pageNumber: Int
     $sortBy: String
     $sortDirection: SortDirection
+    $searchTerm: String
```
Comment (nazarPryt) [inline src/entities/paymentsList/api/getAllPayments.api.ts:8]: sda sdfg d g

Comment (gsricco) [reply src/entities/paymentsList/api/getAllPayments.api.ts:8]: ok

File: src/entities/paymentsList/api/getAllPayments.api.types.tsx [hunk 3]
```diff
@@ -82,6 +90,7 @@ export const GetAllPaymentsDocument = gql`
  *      pageNumber: // value for 'pageNumber'
  *      sortBy: // value for 'sortBy'
  *      sortDirection: // value for 'sortDirection'
```
Comment (nazarPryt) [inline src/entities/paymentsList/api/getAllPayments.api.types.tsx:92]: start review

File: src/entities/paymentsList/api/getAllPayments.api.types.tsx [hunk 4]
```diff
@@ -38,12 +44,14 @@ export const GetAllPaymentsDocument = gql`
     $pageNumber: Int
     $sortBy: String
     $sortDirection: SortDirection
+    $searchTerm: String
   ) {
-    getAllPayments(
+    getPayments(
```
Comment (gsricco) [inline src/entities/paymentsList/api/getAllPayments.api.types.tsx:49]: dfsfsfdf

File: src/entities/paymentsList/api/getAllPayments.api.types.tsx [hunk 5]
```diff
@@ -7,21 +7,27 @@ const defaultOptions = {} as const
...
-      currency: Types.CurrencyType
-      id: number
+      amount?: null | number
+      avatars?: Array<{
+        __typename?: 'Avatar'
+        height?: null | number
+        url?: null | string
+        width?: null | number
+      }> | null
+      createdAt?: any | null
```
Comment (gsricco) [inline src/entities/paymentsList/api/getAllPayments.api.types.tsx:28]: commit

File: src/entities/postsList/api/getPosts.api.types.tsx (hunk 1 above)
Comment (gsricco) [reply src/entities/postsList/api/getPosts.api.types.tsx:30]: ddddd

File: src/entities/paymentsList/api/getAllPayments.api.types.tsx (hunk 5 above)
Comment (nazarPryt) [reply src/entities/paymentsList/api/getAllPayments.api.types.tsx:28]: kjhkjh

**Review by nazarPryt — APPROVED** [review]
approved comment

**Pull request merged**